### PR TITLE
lth and htl a/b testing

### PR DESCRIPTION
**Pull request by Pomax**
bypasses IE11, due to https://github.com/mozilla/donate.mozilla.org/issues/2097

### CODE REVIEW

File: src/lib/queryParser.js [hunk 1]
```diff
@@ -29,11 +29,11 @@ module.exports = function(queryString, locale) {
     presets = currency.presets[frequency];
   }
 
-  if (test && toString.call(test) === "[object Array]") {
+  if (test && test.join && test.map) {
```
Comment (alanmoo) [inline src/lib/queryParser.js]: Is this effectively a browser compatibility check?

File: src/pages/one-page.js [hunk 2]
```diff
@@ -3,6 +3,10 @@ import MozillaFooter from '../components/mozilla/footer.js';
 import Header from '../components/header.js';
 import SingleForm from '../components/single-form.js';
 
+// NOTE: The main page is currently about.js, due to
```
Comment (alanmoo) [inline src/pages/one-page.js]: Where's this from?

File: src/lib/queryParser.js (hunk 1 above)
Comment (Pomax) [reply src/lib/queryParser.js]: the `test.map` is probably not even required. The code that it wraps is for "if this was an array, turn it into a string", so just testing for "does `test.join` exist?" is probably sufficient.

File: src/pages/one-page.js (hunk 2 above)
Comment (Pomax) [reply src/pages/one-page.js]: from @ScottDowne

File: src/lib/queryParser.js (hunk 1 above)
Comment (Pomax) [reply src/lib/queryParser.js]: removed test.map

File: src/pages/one-page.js [hunk 3]
```diff
@@ -12,7 +16,15 @@ var onePage = React.createClass({
     return (
       <div className={className}>
         <Header alt={this.context.intl.formatMessage({id: 'donate_to_mozilla'})}></Header>
-        <SingleForm/>
+        <SingleForm
```
Comment (alanmoo) [inline src/pages/one-page.js]: Why did this suddenly get lots o' props?

Comment (Pomax) [reply src/pages/one-page.js]: because it was missing, and so it was just showing whatever magical defaults were baked in.

**Pull request merged**